docs/variants/msi_z690/firmware-update.md

```diff
@@ -12,6 +12,46 @@ sudo dmidecode -t bios | grep Version
 
 Alternatively, it can be checked in the `BIOS Setup Menu`.
 
+## Prerequisites
+
+Before proceeding, ensure that the firmware protections are disabled in
+[Dasharo Security Options](../..//dasharo-menu-docs/dasharo-system-features.md).
+Both `BIOS boot medium lock` and `Enable SMM BIOS write protection` should be
+unchecked. [UEFI Secure Boot](../../dasharo-menu-docs/device-manager.md#secure-boot-configuration)
+must be disabled as well (uncheck `Attempt Secure Boot` if
+`Current Secure Boot State` does not say `Disabled`). To apply changes you will
+need to reboot.
+
+You can use [Dasharo Tools Suite (DTS)](../../dasharo-tools-suite/releases.md)
+or your own Linux distribution for firmware update. You will need a correct
+flashrom application for that purpose. Linux distributions may not yet have the
```

Comment:
how to identify if I have correct flashrom version?

Reply:
If one follows our guide, it gets the right flashrom. I ask to build it from our sources in subsequent sentences

Reply:
Refactored the sections a lot. Please review again